features/attribute_browse.feature

```diff
@@ -0,0 +1,11 @@
+#language: fr
+
+Fonctionnalité: Sélectionner un item selon un attribut donné
+
+Scénario :
+
+  Soit 'ABT' est affiché
```

Comment:
Cette étape teste la page affichée. Mais du coup qu'elle est la page affichée ?
Il manque une toute première étape de "visite" d'une URI. Vous en avez déjà un certain nombre qui sont implémentées : https://github.com/Hypertopic/Porphyry/blob/d5599c5512140df07d4be7a19619ff8715c03c86/features/step_definitions/context.rb

La plus simple serait : 
```cucumber
Soit "vitraux" le portfolio ouvert
```
Cependant ce n'est pas une très bonne idée, car le chargement de la page risque de dépasser le temps de timeout. C'est pourquoi, les [autres scénarios](https://github.com/Hypertopic/Porphyry/blob/d5599c5512140df07d4be7a19619ff8715c03c86/features/portfolio_browse.feature) utilisent par exemple : 

```cucumber
Soit "Abram/Abraham" les rubriques sélectionnées
```

Du coup, il faudra changer en conséquence les autres données (attention, de plus, le serveur Argos de test a 200 fois moins de données que le serveur de production, choisissez autant que possible des données qui y sont déjà).

Reply:
J'ai vue que dans une autre pull request sur le ticket 158 que vous aviez conseillé de renommer le fichier en portfolio_search.feature, est-ce nécessaire ici aussi ? Vous préciser d'ajouter aussi "pending" et "to be implemented" si la fonctionnalité n'est pas encor eimplémentée, à quel endroit dois-je ajouter ces termes?

Reply:
@benel

Reply:
@benel Pouvez-vous nous communiquer une réponse à la question de FirielEinor s'il vous plait ? Merci d'avance

Reply:
> J'ai vue que dans une autre pull request sur le ticket 158 que vous aviez conseillé de renommer le fichier en portfolio_search.feature, est-ce nécessaire ici aussi ?

Oui, effectivement. Pour les mêmes raisons.

> Vous préciser d'ajouter aussi "pending" et "to be implemented" si la fonctionnalité n'est pas encor eimplémentée, à quel endroit dois-je ajouter ces termes ?

Je le mettrais en tant que première ligne du test (suivant les choix d'implémentation des outils de test, soit ils font le test et n'en tiennent pas compte, soit ils ne le font pas du tout). La syntaxe est `pending "Message à afficher"` (ici le message c'est "To be implemented").